cp3pt0-deployment/setup_singleton.sh

```diff
@@ -231,7 +231,15 @@ function is_migrate_licensing() {
     fi
 
     title "Check migrating LTSR ibm-licensing-operator"
-    wait_for_licensing_instance_deployment
+    # wait for ibm-licensing-operator instance 
+    local licensing_operator_exist=$("$OC" get clusterserviceversion.operators.coreos.com -A | (grep ibm-licensing-operator || echo "fail"))
```

Comment:
I am thinking if checking the deployment here is better than CSV? In case we hit the known OLM issue with unbounded CSV and sub?

```
local licensing_operator_exist=$("$OC" get deployment -A | (grep ibm-licensing-operator || echo "fail"))
```